Source/WebCore/dom/ContainerNode.cpp

```diff
@@ -718,7 +725,9 @@ void ContainerNode::replaceAll(Node* node)
     NodeVector removedChildren;
     removeAllChildrenWithScriptAssertion(ChildChange::Source::API, removedChildren, DeferChildrenChanged::Yes);
 
-    executeNodeInsertionWithScriptAssertion(*this, *node, nullptr, ChildChange::Source::API, ReplacedAllChildren::Yes, [&] {
+    ReplacedAllChildren replacedAllChildren = is<Element>(*node) || removedChildren.containsIf([](auto& node) { return is<Element>(node); }) ? ReplacedAllChildren::YesIncludingElements : ReplacedAllChildren::YesNotIncludingElements;
```

Comment:
This isn't great. Iterating over each child again can end up messing up TLB entries and such.